memdb/memdb.go

```diff
@@ -698,6 +698,31 @@ func (db *MemDb) NewestSha() (*btcwire.ShaHash, int64, error) {
 	return &blockSha, int64(numBlocks - 1), nil
 }
 
+// FetchAddrIndexTip isn't currently implemented. This is a part of the
+// btcdb.Db interface implementation.
+func (db *MemDb) FetchAddrIndexTip() (*btcwire.ShaHash, int64, error) {
+	return nil, 0, nil
```

Comment:
Rather than returning nil for no error here, I'd rather have it return `ErrNotImplemented`.